node/src/main/scala/com/wavesplatform/settings/BlockchainSettings.scala

```diff
@@ -238,12 +239,14 @@ object BlockchainSettings {
         val rewardsSettings        = config.as[RewardsSettings](s"custom.rewards")
         (addressSchemeCharacter, functionalitySettings, genesisSettings, rewardsSettings)
     }
+    val useEvaluatorV2 = config.getAs[Boolean]("use-evaluator-v2").getOrElse(false)
```

Comment:
```suggestion
    val useEvaluatorV2 = false
```
The code which actually reads the value from config should only be present in your private branch

Reply:
Fixed in https://github.com/wavesplatform/Waves/pull/3092